Makefile

```diff
@@ -27,15 +28,29 @@ pyflakes:
 	find . \( -name _build -o -name var -o -path ./docs \) -type d -prune -o -name '*.py' -print0 | $(XARGS) pyflakes
 
 pycodestyle:
-	find . \( -name _build -o -name var \) -type d -prune -o -name '*.py' -print0 | $(XARGS) -n 1 pycodestyle --repeat --exclude=build/*,docs/*,setup.py --ignore=E731,E402,E231
+	find . \( -name _build -o -name var \) -type d -prune -o -name '*.py' -print0 | $(XARGS) -n 1 pycodestyle --repeat --exclude=build/*,docs/*,setup.py --ignore=E731,E402,E231,E203
```

Comment:
This rule is about spaces around the `:`.
black format -> `serial.write(command_bytes[i : min(i + 32, len(command_bytes))])`
pycodestyle  -> `serial.write(command_bytes[i: min(i + 32, len(command_bytes))])`